lib/http_perform.c

```diff
@@ -303,6 +304,7 @@ sf_bool STDCALL http_perform(CURL *curl,
                       curl_easy_strerror(res));
             break;
         }
+#endif
```

Comment:
@sfc-gh-zli and @sfc-gh-smirzaei You comment this code for your XP builds. So provided a flag to skip this compilation when building for XP. And you can use this macro to avoid things not needed for XP.
`#ifndef LIBSFCLI_FOR_XP`